src/block_cache/block_io.c

```diff
@@ -260,6 +261,75 @@ __wt_blkcache_read(WT_SESSION_IMPL *session, WT_ITEM *buf, WT_PAGE_BLOCK_META *b
     return (ret);
 }
 
+static int
```

Comment:
Add comment for the function?

Reply:
Done in d71aeb75b834b222165837479b34010bbaf3ef7e